e7-damage-calc/form.js

```diff
@@ -773,7 +773,7 @@ $(() => {
     const heroSelector = document.getElementById('hero');
     const artiSelector = document.getElementById('artifact');
     Object.keys(heroes).map((id => {
-      $(heroSelector).append(`<option value="${id}" data-content="${elemIcon(heroes[id].element)}${classIcon(heroes[id].classType)} <span>${heroName(id)}</span>">${heroName(id)}</option>`)
+      $(heroSelector).append(`<option value="${id}" data-tokens="${heroName(id)} ${heroNickname(id)}" data-content="${elemIcon(heroes[id].element)}${classIcon(heroes[id].classType)}<span>${heroName(id)}</span>">${heroName(id)}</option>`)
```

Comment:
```suggestion
      $(heroSelector).append(`<option value="${id}" data-tokens="${heroNickname(id)}" data-content="${elemIcon(heroes[id].element)}${classIcon(heroes[id].classType)}<span>${heroName(id)}</span>">${heroName(id)}</option>`)
```
I don't think we need to duplicate the name since it's already the `<option>` content